app/views/ReportsView/index.tsx

```diff
@@ -17,43 +26,44 @@ const ReportsView = (): JSX.Element => {
   } = useResource();
   const [rowData, setRowData] = useState<ReportType[]>();
 
+  let statesArray = allStates;
+  if (!nonproductionAccess) {
+    statesArray = statesArray.filter((elem) => !nonproductionStates.includes(elem));
+  }
+  if (!unreviewedAccess) {
+    statesArray = statesArray.filter((elem) => !unreviewedStates.includes(elem));
+  }
+  const states = statesArray.join(',');
+  const reportsQuery = useReports(states);
```

Comment:
would destructure this into 

`const { isLoading: isApiLoading, data } = useReports(states);`

instead of checking 
`reportsQuery.isLoading;` and 
`reportsQuery.data;`